core/.js/src/main/scala/io/udash/view/ViewRenderer.scala

```diff
@@ -61,7 +62,7 @@ private[udash] class ViewRenderer(rootElement: => Element) {
       case c: ContainerView => c.renderChild(None)
       case _ =>
     }
-    views.trimEnd(views.length - unmodifiedViews)
+    views.trimEnd(views.length - unmodifiedViews): @nowarn
```

Comment:
why not `dropRightInPlace` instead?

Reply:
If I'm not missing anything there's no easy way to provide it for 2.12 without making things ugly for 2.13.